cachito/web/api_v1.py

```diff
@@ -790,7 +790,7 @@ def get_request_metrics_summary():
     """Return a summary about completed requests for a given period of time."""
     args = RequestMetricsSummaryArgs(**flask.request.args)
     finished_to = args.finished_to
-    if isinstance(finished_to, date):
+    if type(finished_to) is date:
```

Comment:
I'm afraid that somebody will change it, when a linter will report this a not pythonic and suggest to use isinstance. Can we just use `if not isinstance(finished_to, datetime)` ?